src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1568,6 +1568,32 @@ void Pane::Restore(std::shared_ptr<Pane> zoomedPane)
     }
 }
 
+uint16_t Pane::GetPaneId() noexcept
+{
+    return _paneId;
+}
+
+void Pane::SetPaneId(uint16_t id) noexcept
+{
+    _paneId = id;
+}
+
+void Pane::FocusPaneWithId(const uint16_t id)

```

Comment:
```suggestion
void Pane::FocusPane(const uint16_t id)
```
and add a comment saying what the param is?